### ISSUE TITLE
Checkbox Hide activity breakdown and Non Billables from Reporting

### ISSUE THREAD

**Issue by Crumar** [kimai/kimai#3519]
**Is your feature request related to a problem? Please describe.**
Personally, I use Kimai to track my Working Time for my company only.
My company is not entitled to in depth breakdowns of this, but I maintain this for personal reasons and discussions.
However, I need to enter durations worked on projects into our internal tool, which is only on project level. This creates a lot of visual moving between the reporting chart.

I only use Weekly view for one user, but potentially others would benefit on all pages.

**Describe the solution you'd like**
I'd like two options:

1. Checkbox Show/Hide Activity Breakdown
2. Dropdown Billable Yes / No / All

First Checkbox modifies view to only show projects, and on demand the activity breakdown.

Second Dropdown includes the matching records in the aggregates.

**Comment by morja** [comment]
Id'd love this feature as well. And related to this: In the dashboard the yearly revenue does not include non-billable records. But the yearly view in reporting does include it, and there is no option to exclude it. As the non-billable entries, are actually no revenue, but only an internal tracking of costs, it would really be preferable to either exclude them by default or make it an option as suggested in this issue.

Thanks a lot for the great work. I really enjoy using Kimai!

**Comment by kevinpapst** [comment]
This?

<img width="786" alt="Bildschirmfoto 2024-11-17 um 09 19 39" src="https://github.com/user-attachments/assets/087afa26-5bc0-4a7c-921a-d8c4358f4705">

**Comment by morja** [comment]
Yes, for me the revenue option includes non-billable entries. And the costs show exactly the same as revenue.

**Comment by kevinpapst** [comment]
Here is a random example from my dev system:
<img width="1355" alt="Bildschirmfoto 2024-11-24 um 23 17 59" src="https://github.com/user-attachments/assets/4b065159-69f8-4cdd-9ed4-6fa2a01d3bc5">
<img width="1008" alt="Bildschirmfoto 2024-11-24 um 23 16 31" src="https://github.com/user-attachments/assets/84fe79ec-7316-44c0-8b4e-5b8e93cf0afd">
<img width="898" alt="Bildschirmfoto 2024-11-24 um 23 18 49" src="https://github.com/user-attachments/assets/a225ae04-d91b-4e2f-9fa6-76620d5dac83">

The reports show the correct data, unless I am blind and don't understand what you want to say.

Lets not hijack this feature request for further discussions. 
Please open separate discussions or feature request for everything unrelated to the inital post.

**Comment by morja** [comment]
Maybe I am misunderstanding how it should work...

I just created a dummy to showcase it. I have two activities, one billable, one non-billable. Both show up as revenue.

Should non-billable not be excluded in revenue? It is not revenue that I make as I will never see any payment for non-billable. I guess I could set the amount to 0 for non-billables , but I still like to have the right amount for internal reference.

Also if I set the Customer and the Project to non-billable, any new entries show up in the revenue.

Also those non-billable entries are actually excluded in the revenue in the dashboard. 

![image](https://github.com/user-attachments/assets/d0e6c9c7-9748-4371-8363-4eb5bbbc7f57)

**Comment by kevinpapst** [comment]
Thanks for the clarification, understood now. Will test again. Maybe it was too late yesterday 😁
